src/EFCore.PG/Metadata/Internal/NpgsqlAnnotationNames.cs

```diff
@@ -45,5 +44,8 @@ public static class NpgsqlAnnotationNames
 
         [Obsolete("Replaced by ValueGenerationStrategy.SerialColumn")]
         public const string ValueGeneratedOnAdd = Prefix + "ValueGeneratedOnAdd";
+
+        [Obsolete("Replaced by built-in EF Core support")]
```

Comment:
Makes sense to write a constant in this class with a value like `"Replaced by built-in EF Core support. Use HasComment"`, and reuse it for all `Obsolete` attributes.

Reply:
I changed the obsolete messages to be more informative, but am keeping them separate since the ones on the builder should be much more concise/easy. The others (like this one in NpgsqlAnnotationNames) will likely not be seen by end users anyway.